### ISSUE TITLE
system response distribution types

### ISSUE THREAD

**Issue by rnugent3** [HydrologicEngineeringCenter/HEC-FDA#580]
please make the combo box in the system response curve control "deterministic only"

**Comment by rnugent3** [comment]
this is what I see on the feature/defaultData branch 

![image](https://user-images.githubusercontent.com/25163165/195415339-8ef8a1c3-d103-460e-993e-47bc3e6b6a38.png)

**Comment by codymccoy** [comment]
Ah, you are in edit mode. I bet i only set it when creating new. I'm on it.

**Comment by codymccoy** [comment]
@rnugent3  I tested in edit mode and I only see the deterministic option. This isn't a new change. It should have been pushed into main a long time ago. Not sure how you are getting all the options. Oh, maybe you are using a really old study? It looks to me like after you save it, it will load with whatever options where saved out. That means that if you saved the lateral structure before we limited it to deterministic, it will show up with all the options.
